lib/strong_migrations/migration.rb

```diff
@@ -186,6 +203,30 @@ def backfill_code(table, column, default)
       end
     end
 
+    def manual_add_foreign_key_code(from_table, to_table)
+      <<~CODE.strip
+        ALTER TABLE #{from_table} ADD CONSTRAINT fk_#{from_table}_#{to_table} FOREIGN KEY (#{to_table.to_s.singularize}_id) REFERENCES #{to_table} (id) NOT VALID;
+      CODE
+    end
+
+    def manual_validate_foreign_key_code(from_table, to_table)
+      <<~CODE.strip
+        ALTER TABLE #{from_table} VALIDATE CONSTRAINT fk_#{from_table}_#{to_table};
+      CODE
+    end
+
+    def foreign_key_code(from_table, to_table)
+      <<~CODE.strip
+        #{command_str(:add_foreign_key, [from_table, to_table, validate: false])}
+      CODE
+    end
+
+    def validation_code(from_table, to_table)
+      <<~CODE.strip
+        #{command_str(:validate_foreign_key, [from_table, to_table])}
```

Comment:
The interpolation shouldn't be needed. Since all of the commands are one-liners, I think we can get rid of the helper methods as well.